# TODO: define exceptions
def Error(Exception):
    pass